common/authen.js

```diff
@@ -3,7 +3,9 @@
 
     angular.module('chaise.authen', ['chaise.utils', 'chaise.storage'])
 
-    .factory('Session', ['messageMap', 'modalUtils', 'StorageService', '$cookies', '$http', '$interval', '$log', '$q', 'UriUtils', '$window', function (messageMap, modalUtils, StorageService, $cookies, $http, $interval, $log, $q, UriUtils, $window) {
+    .factory('Session', ['messageMap', 'modalUtils', 'StorageService', '$cookies', '$http', '$interval', '$log', '$q', 'UriUtils', 'ConfigUtils', '$window', function (messageMap, modalUtils, StorageService, $cookies, $http, $interval, $log, $q, UriUtils, ConfigUtils, $window) {
+
+        var chaiseConfig = ConfigUtils.getConfigJSON();
```

Comment:
This can be removed and instead, `$rootScope` should be included as a dependency. Then each place that chaiseConfig is being used, it should be called like: `$rootScope.chaiseConfig` without any conditional checks. This should be done everywhere that this was defined like this.